docs/conf.py

```diff
@@ -53,10 +53,8 @@
 
 # General information about the project.
 project = "Pillow (PIL Fork)"
-copyright = (
-    "1995-2011 Fredrik Lundh, 2010-2024 Jeffrey A. Clark (Alex) and contributors"
-)
-author = "Fredrik Lundh, Jeffrey A. Clark (Alex), contributors"
+copyright = "1995-2011 Fredrik Lundh and contributors, 2010-2024 Jeffrey A. Clark and contributors."
```

Comment:
Add new line to fix lint:
```suggestion
copyright = (
    "1995-2011 Fredrik Lundh and contributors, "
    "2010-2024 Jeffrey A. Clark and contributors."
)
```